src/main/java/com/scouts/kitchenplanerbackend/repositories/recipes/RecipeRepository.java

```diff
@@ -34,4 +42,32 @@ public interface RecipeRepository extends JpaRepository<RecipeEntity, Long> {
 
     @Query("select r from RecipeEntity r")
     RecipeEntityDTO getRecipeEntityBy(@Param("id") long id);
+
+    /**
```

Comment:
Make them privat/protected or something like that, because the uri only should be updated through "updateImagePath"